scripts/config/lib/nginx_config_util.rb

```diff
@@ -44,4 +44,16 @@ def self.match_redirects(redirects, uri)
 
     false
   end
+
+  def self.interpolate(string, vars)
+    regex = /\${(\w*?)}/
+
+    string.scan(regex).inject(string) do |acc, capture|
+      var_name = capture.first
+      value = vars[var_name] if vars
+      acc.sub!("${#{var_name}}", value) if value
```

Comment:
@ojacobson the `?` limits it to be non greedy, but using `Regex.escape` we should probably rewrite this to not use a generic regex to scan against.